docs/sidebars.js

```diff
@@ -79,6 +79,10 @@ module.exports = {
       type: 'doc',
       id: 'troubleshooting',
     },
+    {
+      type: 'doc',
+      id: 'upgrade-existing-vclusters',
+    },
```

Comment:
This could be part of the "Operator Guide" section, but I don't feel that strongly about this :)